lib/src/utils/matrix_file.dart

```diff
@@ -266,7 +266,7 @@ class MatrixImageFile extends MatrixFile {
         height: image.height > image.width ? arguments.maxDimension : null,
         width: image.width >= image.height ? arguments.maxDimension : null);
 
-    final encoded = encodeNamedImage(resized, arguments.fileName);
+    final encoded = encodeNamedImage(arguments.fileName, resized);
```

Comment:
why we need it?

Reply:
i don't focus on this, it is an error when reset to the current matrix version 0.18.0, so i have to change it that way.